### PR TITLE
Reduce ansible/ansible README to essentials

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
A link to the Working Groups was missing. I also reduced the overall amount of information on that first introduction page of our project.

What information would you share with someone that does not know the project already ?
A lot of the information is not essential and covered elsewhere.
In fact I think we can reduce it even more...

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
README

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,23 +5,21 @@ Ansible
...
-trivializing things like zero-downtime rolling updates with load
-balancers.
+configuration management, application deployment, cloud provisioning,
+ad-hoc task execution, and multi-node orchestration. Ansible makes complex
+changes like zero-downtime rolling updates with load balancers easy. More information on `the Ansible website <https://ansible.com/>`_.
 
-Read the documentation and more at https://ansible.com/
+You can install a released version of Ansible via ``pip``, a package manager, or
+our `release repository <https://releases.ansible.com/ansible/>`_. See our
+`installation guide <https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>`_ for details on installing Ansible
```
Comment (webknjaz) [inline README.rst]: Can we multiline this to make lines shorter?

Comment (webknjaz) [reply README.rst]: maybe use detached ref mappings?

File: README.rst [hunk 2]
```diff
@@ -5,23 +5,21 @@ Ansible
...
-Most users should probably install a released version of Ansible from ``pip``, a package manager or
-our `release repository <https://releases.ansible.com/ansible/>`_. `Officially supported
-<https://www.ansible.com/ansible-engine>`_ builds of Ansible are also available. Some power users
-run directly from the development branch - while significant efforts are made to ensure that
-``devel`` is reasonably stable, you're more likely to encounter breaking changes when running
-Ansible this way.
+Power users and developers can run the development
+branch directly. It is reasonably stable, but you are more likely to encounter
+breaking changes when running the ``devel`` branch. We recommend participating
+in the Ansible community if you want to run the development branch - see Get Involved below.
```
Comment (webknjaz) [inline README.rst]:
```suggestion
in the Ansible community if you want to run the development branch — see Get Involved below.
```

Comment (webknjaz) [reply README.rst]: "Get Involved" could be referenced using a relative anchor

File: README.rst [hunk 3]
```diff
@@ -47,51 +45,35 @@ Get Involved
    kinds of ways to contribute to and interact with the project,
    including mailing list information and how to submit bug reports and
    code to Ansible.
-*  All code submissions are done through pull requests to the ``devel`` branch.
-*  Feel free to talk to us before making larger changes
+*  Join a `Working Group
+   <https://github.com/ansible/community/wiki>`_, an organized community devoted to a specific technology domain or platform.
```
Comment (webknjaz) [inline README.rst:52]: Please split this line
```suggestion
   <https://github.com/ansible/community/wiki>`_, an organized community
   devoted to a specific technology domain or platform.
```

File: README.rst [hunk 4]
```diff
@@ -47,51 +45,35 @@ Get Involved
...
+*  Submit proposed a code update through a pull request to the ``devel`` branch.
```
Comment (webknjaz) [inline README.rst:53]: probably worth linking to https://help.github.com/en/articles/about-pull-requests

File: README.rst [hunk 5]
```diff
@@ -47,51 +45,35 @@ Get Involved
...
+Based on team and community feedback, an initial roadmap will be published for a major or minor version (ex: 2.7, 2.8).
 The `Ansible Roadmap page <https://docs.ansible.com/ansible/devel/roadmap/>`_ details what is planned and how to influence the roadmap.
 
 Authors
 =======
 
 Ansible was created by `Michael DeHaan <https://github.com/mpdehaan>`_
-(michael.dehaan/gmail/com) and has contributions from over 4000 users
-(and growing). Thanks everyone!
+and has contributions from over 4000 users (and growing). Thanks everyone!
```
Comment (webknjaz) [inline README.rst:79]:
```suggestion
and has contributions from over 4200 users (and growing). Thanks, everyone!
```

File: README.rst [hunk 6]
```diff
@@ -5,23 +5,21 @@ Ansible
 *******
 
 Ansible is a radically simple IT automation system. It handles
-configuration-management, application deployment, cloud provisioning,
-ad-hoc task-execution, and multinode orchestration -- including
-trivializing things like zero-downtime rolling updates with load
-balancers.
+configuration management, application deployment, cloud provisioning,
```
Comment (liquidat) [inline README.rst:8]: How about we include network automation and maybe even devops here as well? After all, network automation is a huge topic with Ansible, and many who read "configuration management" will fail to see that this includes network devices.

File: README.rst (hunk 2 above)
Comment (acozine) [reply README.rst]: Do relative anchors show up on GitHub?

File: README.rst (hunk 4 above)
Comment (gundalow) [reply README.rst:53]: Maybe add link to docs.ansible.com/ansible/latest/community

File: README.rst (hunk 5 above)
Comment (acozine) [reply README.rst:79]: For the sake of long-term maintenance, I'd like to stick with thousands here - if we update for every 100 new contributors, we'll have to check the README pretty often to keep up.

Comment (webknjaz) [reply README.rst:79]: @acozine I can write a bot for that :laughing:

File: README.rst (hunk 2 above)
Comment (webknjaz) [reply README.rst]: @acozine I believe so

File: README.rst [hunk 7]
```diff
@@ -5,23 +5,21 @@ Ansible
...
+ad-hoc task execution, and multi-node orchestration. Ansible makes complex
+changes like zero-downtime rolling updates with load balancers easy. More information on `the Ansible website <https://ansible.com/>`_.
```
Comment (jamescassell) [inline README.rst:10]: 'rolling updates with load balancers' seems like a very specific use case, but maybe it's the best example

**Review by jamescassell — APPROVED** [review]
shipit

File: README.rst (hunk 1 above)
Comment (gundalow) [reply README.rst]: If it passes CI it's fine.

File: README.rst (hunk 7 above)
Comment (gundalow) [reply README.rst:10]: I think this is OK. It's a good example of something that's more complex.

**Review by gundalow — APPROVED** [review]
This is a good improvement over what we have, thank you.

I feel like this could continually be tweaked.

If @acozine (Docs Lead) is happy then this can be merged

File: README.rst (hunk 6 above)
Comment (acozine) [reply README.rst:8]: Addressed by 1f59d0c0118c11a56cb1a4d760b176290287e12e.

File: README.rst (hunk 2 above)
Comment (acozine) [reply README.rst]: I changed the order of the sections to address this concern without an internal link. See 1f59d0c0118c11a56cb1a4d760b176290287e12e.

**Comment by gundalow** [comment]
Thank you everybody that worked on this!

**Pull request merged**